File: typescript/easy/2469.convertTemperature.ts
function convertTemperature(celsius: number): number[] {
    let k: number = celsius + 273.15
    let f: number = celsius * 1.8 + 32.0

    return [k, f]
}

console.log(convertTemperature(36.5)) // [309.65000, 97.70000]
// Explanation: Temperature at 36.50 Celsius :
// converted in Kelvin is 309.65
// converted in Fahrenheit is 97.70.

File: python/easy/2469.convertTemperature.py
from typing import List

class Solution:
    def convertTemperature(self, celsius: float) -> List[float]:
      kelvin:float = celsius + 273.15
      fahrenheit:float = celsius * 1.80 + 32.00

      return [kelvin,fahrenheit]
        
        
if __name__ == "__main__":
    solution = Solution()
    print(solution.convertTemperature(36.50)) # [309.65000,97.70000]
    print(solution.convertTemperature(122.11)) # [395.26000,251.79800]
